aries-site/src/examples/templates/wizard/WizardValidationExample.js

```diff
@@ -0,0 +1,346 @@
+import React, { useState, useContext } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  CheckBoxGroup,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Layer,
+  List,
+  RadioButtonGroup,
+  ResponsiveContext,
+  Select,
+  Text,
+  TextArea,
+  TextInput,
+} from 'grommet';
+import {
+  Checkmark,
+  FormClose,
+  FormNextLink,
+  FormPreviousLink,
+} from 'grommet-icons';
+
+const defaultFormValues = {
+  'text-input-validation': '',
+  'radio-button-group-validation': '',
+  select: '',
+  checkboxgroup: '',
+  'text-area': '',
+};
+
+const stepOneValidate = values => {
+  const emailRegex = RegExp(
+    '[^@ \\t\\r\\n]+@[^@ \\t\\r\\n]+\\.[^@ \\t\\r\\n]+',
+  );
+  const emailValid = emailRegex.test(values['text-input-validation']);
+
+  return {
+    email: emailValid ? '' : 'Invalid email address.',
+    isValid: !!emailValid,
+  };
+};
+
+const StepOne = ({ active, setActive, formValues }) => {
+  const [error, setError] = useState({
+    email: '',
+    radiobuttongroup: '',
+  });
+
+  return (
+    <>
+      <Box margin={{ bottom: 'large' }}>
+        <FormField
+          label="Email"
+          htmlFor="text-input-validation"
+          name="text-input-validation"
+          error={error.email}
+          onChange={() => setError(stepOneValidate(formValues))}
+        >
+          <TextInput
+            placeholder="jane.smith@hpe.com"
+            id="text-input-validation"
+            name="text-input-validation"
+            type="email"
+          />
+        </FormField>
+        <FormField
+          help="Click on the ‘eye’ icon for the description text field to hide."
```

Comment:
I am not sure I understand this message.

Reply:
This is coming from the designs, but I'm reading it now and agree it's unclear what is supposed to happen here. I am not sure any help text is necessary for this field.